test/e2e/mock-e2e.js

```diff
@@ -117,14 +122,26 @@ async function setupMocking(
 ) {
   const privacyReport = new Set();
   await server.forAnyRequest().thenPassThrough({
-    beforeRequest: (req) => {
-      const { host } = req.headers;
+    beforeRequest: ({ headers: { host }, url }) => {
       if (blacklistedHosts.includes(host)) {
         return {
           url: 'http://localhost:8545',
         };
+      } else if (
+        WHITE_LISTED_URLS.includes(url) ||
+        WHITE_LISTED_HOSTS.includes(host)
+      ) {
+        // If the URL is whitelisted, we pass the request as it is, to the live server.
+        console.log('Request going to a live server ============', url);
+        return {};
       }
-      return {};
+      console.log('Request redirected to a catch all mock ============', url);
+      return {
+        // If the URL is not whitelisted nor blacklisted, we redirect the request to the localhost fixture server
+        // to always return a 200 response.
```

Comment:
😄 Well that's one way to deal with them.

Perhaps instead of redirecting to this server, we could return the response here directly? The fixture data doesn't seem especially helpful to return here.

We can return an object with a `response` property to respond to the request directly here, without it being passed through further (see https://httptoolkit.github.io/mockttp/interfaces/requestHandlers.CallbackRequestResult.html#response)

Reply:
I'm also curious, why a successful response rather than a failure? Just to minimize disruption?

Maybe this is the best idea as a first step, but eventually it would seem helpful to track these in some way so we can more easily catch them. And even though this is logged, it may be easy to miss (people usually don't read these logs until something fails).

Reply:
I am unsure on the best approach as well, but going with what you've suggested here (a successful 200 response) seems like an OK decision for now. It's easy to change it later if we change our minds.

Reply:
it make sense to me! Maybe we can proceed with this 200 as the first step, then, we'll work on reducing the allowlist to 0 and as a later step, review what to do with the catch-all bucket again